packages/@okta/vuepress-site/docs/guides/customize-themes/index.md

```diff
@@ -37,7 +37,7 @@ The [Brands API](/docs/reference/api/brands/) allows you to set all of the follo
 
 ## Important: Overriding themes in the code template editors
 
-You can customize individual parts of your Okta org experience using the various code editors available in the Admin Console (for example, **Settings** > **Customizations** > **Custom Sign-in** to customize the Okta-hosted sign-in page). However, if you use the Brands API to delete or augment any of the [branding variables](https://help.okta.com/okta_help.htm?type=oie&id=ext-branding-variables), the API calls override any changes you make using the Admin Console.
+You can customize individual parts of your Okta org experience using the various code editors available in the Admin Console (for example, **Customizations** > **Sign-in page code editor** to customize the Okta-hosted sign-in page). However, if you use the Brands API to delete or augment any of the [branding variables](https://help.okta.com/okta_help.htm?type=oie&id=ext-branding-variables), the API calls override any changes you make using the Admin Console.
```

Comment:
Remove "of the" in " .. any of the branding variables .." so it becomes "any branding variables"

Reply:
I've not seen the use of "Important" as part of a heading before, although maybe it's useful to highlight. Perhaps ping Style channels on this. I don't want to hold up PR on this item, though.

Reply:
@brianduffield-okta I'll just remove it. It's legacy content so I'm not too attached to it :)